README.md

```diff
@@ -2048,6 +2053,90 @@ Probe type. Valid values are 'icmp', 'dns', 'tcp', 'udp' or keyword 'default'.
 Weight for traffic distribution. Valid values are Integer, keyword 'default'.
 
 --
+### Type: cisco_itd_service
+
+Manages configuration of ITD (Intelligent Traffic Director) service
+
+| Platform | OS Minimum Version | Module Minimum Version |
+|----------|:------------------:|:----------------------:|
+| N9k      | 7.0(3)I3(1)        | 1.3.0                  |
+| N30xx    | unsupported        | unsupported            |
+| N31xx    | unsupported        | unsupported            |
+| N56xx    | unsupported        | unsupported            |
+| N6k      | unsupported        | unsupported            |
+| N7k      | 7.3(0)D1(1)        | 1.3.0                  |
+| N8k      | unsupported        | unsupported            |
+| IOS XR   | unsupported        | unsupported            |
+
+#### <a name="cisco_itd_service-caveats">Caveats</a>
+
+| Property | Caveat Description |
+|:--------|:-------------|
+| `nat_destination` | Supported only on N7k |
+| `peer_local` | Supported only on N9k |
+| `peer_vdc` | Supported only on N7k |
```

Comment:
I have removed the reference to vrf altogether. You want me to add that back only in the README file?